ledger/indexer-benchmark/src/main/scala/ledger/indexerbenchmark/IndexerBenchmark.scala

```diff
@@ -90,7 +89,7 @@ class IndexerBenchmark() {
               .migrateOnly(
                 jdbcUrl = config.indexerConfig.jdbcUrl
               )
-              .map(_ => indexerFactory.initialized())(indexerEC),
+              .map(_ => indexerFactory.initialized(loggingContext))(indexerEC),
```

Comment:
Was this change necessary? `loggingContext` was previously used as an implicit value.

Reply:
Sadly it is necessary. Won't compile otherwise :shrug: